iris/src/iris/common/tutor_suggestion.py

```
@@ -0,0 +1,258 @@
+    return "No artifact found in chat history."
+
+
+def get_chat_history_without_user_query(chat_history: List[PyrisMessage]) -> str:
+    """
+    Extracts the chat history without the user query.
+    :param chat_history: List of messages in the chat history.
+    :return: The chat history as a string.
+    """
+    chat_history_str = "No chat history found."
+    if chat_history:
+        if chat_history[-1].sender == IrisMessageRole.USER:
```

Ruff (0.12.2)

Use a single `if` statement instead of nested `if` statements

Combine `if` statements using `and`

(SIM102)